P/PROJ/build_tarballs.jl

```diff
@@ -90,8 +87,8 @@ dependencies = [
     # Host SQLite needed to build proj.db
     HostBuildDependency("SQLite_jll")
     Dependency("SQLite_jll")
-    Dependency("Libtiff_jll")
-    Dependency("LibCURL_jll"; compat="7.73")
+    Dependency("Libtiff_jll"; compat="~4.5.1")
+    Dependency("LibCURL_jll")
```

Comment:
Curl must have colpa bounds: https://github.com/JuliaPackaging/Yggdrasil/issues/3024

Reply:
Should I leave it as `compat="7.73"`, or is the `compat="7.73,8"` I set in cd47fa86b2f47ca0d51472bbd9b9dd2e0770384b also ok?

Reply:
Well, you to allow v8 if you want to make this usable in newer versions

Reply:
Doesn't `compat="7.73,8"` allow v8? Not sure what to put here.

Reply:
Yes, I was answering the previous question.